File: Gcd.java
import java.util.Scanner;
public class Gcd{
    public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    System.out.print("Enter two numbers of choice: ");
    int firstNumber = input.nextInt();
    int secondNumber = input.nextInt();

    int count = 0;
    int counter = 0;
    int largest = 0;
    for(count = 1; count < firstNumber ; count++)
    if(firstNumber % count == 0 && secondNumber % count == 0){
    
        System.out.println(count + " ");
        if(largest < count) 
           largest = count;
    }System.out.println("The greatest common divisor is "+ largest);

   


}

    





}

File: gcd.js
let first_number = 24
let second_number = 20
let count = 0
let counter = 0
let largest = 0
    for(count = 1; count < first_number ; count++)
        if(first_number % count == 0 && second_number % count == 0){
    console.log(count)
    counter=count
    }
console.log("The greatest divisor is ", counter)

